jupyter_book/commands/__init__.py

```diff
@@ -225,6 +225,36 @@ def toc(path, filename_split_char, skip_text, output_folder):
     _message_box(f"Table of Contents written to {output_file}")
 
 
+@main.command()
+@click.argument("path-output")
+@click.option("-a", "--all", "all_", is_flag=True, help="Remove build directory.")
```

Comment:
I am curious what the `all_` pattern means here - does it mean people could do something like `jupyter-book clean all_`?

Reply:
Well `all` is a python function so this is a workaround for that. So, we can do something like `jb clean --all book/`, `jb clean -a book/`, `jb clean book/ --all`, `jb clean book/ -a`. I used the idea that's used in Anaconda to remove environments, which is something like `conda env remove --all`.